suites/google/googleServiceAccountRemovalTest.js

```diff
@@ -25,6 +25,7 @@ Feature('GoogleServiceAccountRemoval');
 // IAM access needed by the monitor service account
 const monitorRoles = [
   'roles/resourcemanager.projectIamAdmin',
+  'roles/iam. serviceAccountAdmin',
```

Comment:
```suggestion
  'roles/iam.serviceAccountAdmin',
```
i didn't check what's the role ID but i don't think there would be a space :p